client/src/store/modules/user/user.js

```diff
@@ -120,22 +120,32 @@ export default {
         context.commit('SET_TOKEN_REFRESH_INTERVAL', null);
       }
 
-      const response = await fetch(makeURL('/api/v1/auth/logout'), {
-        method: 'POST',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify({
-          session_id: context.rootGetters.INTERNAL_UUID,
-        }),
-      });
       await context.commit('SET_AUTH_TOKEN', null);
       await context.commit('SET_CURRENT_USER', null);
       await context.commit('SET_CURRENT_RBAC', null);
       await context.commit('CLEAR_WS_AUTHENTICATION');
-      if (response.ok) {
-        Vue.$toast.success('Successfully logged out!');
+
+      if (context.getters.AUTH_TOKEN) {
```

Comment:
This check is always going to be false, since the token is being set to null before this line.